cmd/destroy.go

```diff
@@ -35,69 +35,62 @@ var destroyCmd = &cobra.Command{
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
+		opts := []clab.ClabOption{
+			clab.WithDebug(debug),
+			clab.WithTimeout(timeout),
+			clab.WithRuntime(rt, debug, timeout, graceful),
+		}
+
+		topos := map[string]struct{}{}
+
 		switch {
 		case !all:
-			// stop if not topo file provided and not all labs are requested
-			// to be deleted
-			if err = topoSet(); err != nil {
-				return err
+			{
+				// stop if not topo file provided and not all labs are requested
+				// to be deleted
+				if err = topoSet(); err != nil {
```

Comment:
I think that is not needed anymore, as when you do `c := clab.NewContainerLab(opts...)` later, the NetContainerlLab will return an error since the topo file is not found

Reply:
removed the func now along with the remaining reference from cmd/deploy.go